app/javascript/components/journey/Contribution.tsx

```diff
@@ -39,7 +39,7 @@ export const Contribution = ({
           ) : (
             <div className="generic">Generic</div>
           )}
-          <time dateTime={awardedAt}>{fromNow(awardedAt)}</time>
+          <time dateTime={earnedOn}>{fromNow(earnedOn)}</time>
```

Comment:
Can you double check this works on now that it's being passed a date, not a datetime pls. Like, it doesn't think that the date is midnight or something.

(I made `earned_on` a date, because it makes the searching faster)

Reply:
So `earned_on` being a date has the result that if you set it to the current time, it will display "X hours ago", where X is the number of hours today. It would be nicer if it would say "today", "yesterday", "x days ago", "x weeks ago", "x months ago" and "x years ago". Is that something you'd like me to look at?

Reply:
Yes please :)

Reply:
Maybe make a new `fromNowDate()` so we can use that explicitly?

Reply:
Will do